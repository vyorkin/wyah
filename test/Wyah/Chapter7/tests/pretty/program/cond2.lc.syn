Program [Decl "eqzero" (ELam (Var "x") (EIf (EOp Eq (EVar (Var "x")) (ELit (LInt 0))) (EVar (Var "True")) (EVar (Var "False")))),Decl "notbool" (ELam (Var "x") (EIf (EVar (Var "x")) (EVar (Var "False")) (EVar (Var "True"))))]
