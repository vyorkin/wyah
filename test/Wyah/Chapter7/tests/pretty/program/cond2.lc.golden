let eqzero = \x -> if x == 0 then True else False;
let notbool = \x -> if x then False else True;